photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -221,6 +226,7 @@ object GameTrainingDriver extends GameDriver {
     setDefault(timeZone, Constants.DEFAULT_TIME_ZONE)
     setDefault(ignoreThresholdForNewModels, false)
     setDefault(incrementalTraining, false)
+    setDefault(savePerGroupEvaluationResult, true) //TODO: Testing purpose, default to false before push
```

Comment:
remove comment

Reply:
This is for testing purpose. The default value should be false. Will remove before push.